### PR TITLE
chore: Update Digital Team Files + clear redirects

### PR DESCRIPTION
**Pull request by hashibot-web**
# Description

Updates managed website-related files.

- `website/Makefile`
- `website/scripts/should-build.sh`
- `website/scripts/website-build.sh`
- `website/scripts/website-start.sh`

> **Note**
>
> This pull request was opened via the Digital team's automation. If you have questions, please tag @hashicorp/web-platform

### CODE REVIEW

File: website/scripts/should-build.sh [hunk 1]
```diff
@@ -9,5 +14,5 @@ if [[ "$VERCEL_GIT_COMMIT_REF" == "stable-website"  ]] ; then
   exit 1;
 else
   # Check for differences in the website directory
-  git diff --quiet HEAD^ HEAD ./ ../internal/gen/controller.swagger.json
-fi
+  git diff --quiet HEAD^ HEAD ./
```
Comment (thiskevinwang) [inline website/scripts/should-build.sh:17]:
```suggestion
  git diff --quiet HEAD^ HEAD ./ ../internal/gen/controller.swagger.json
```

File: website/scripts/should-build.sh [hunk 2]
```diff
@@ -10,4 +15,4 @@ if [[ "$VERCEL_GIT_COMMIT_REF" == "stable-website"  ]] ; then
 else
   # Check for differences in the website directory
   git diff --quiet HEAD^ HEAD ./ ../internal/gen/controller.swagger.json
```
Comment (thiskevinwang) [inline website/scripts/should-build.sh:17]: Note: This line needs to be updated in our TF source template

**Comment by thiskevinwang** [comment]
# Additional Notes

This also clears the `website/redirects.js` file; That has been imported into developer via
- https://github.com/hashicorp/dev-portal/pull/1256/files

File: website/redirects.js [hunk 3]
```diff
@@ -1,315 +1,5 @@
 module.exports = [
   // define your custom redirects within this file.
   // vercel's redirect documentation: https://vercel.com/docs/project-configuration#project-configuration/redirects
-
   // Top level redirect
```
Comment (thiskevinwang) [inline website/redirects.js:4]:
```suggestion
  // example redirect:
  // {
  //   source: '/boundary/docs/some/path',
  //   destination: '/boundary/docs/some/other/path',
  //   permanent: true,
  // },
```

**Review by BRKalow — APPROVED** [review]

**Pull request merged**